src/Temporalio/Runtime.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+
+namespace Temporalio
+{
+    /// <summary>
+    /// Runtime for the Temporal SDK.
+    /// </summary>
+    /// <remarks>
+    /// This runtime carries the internal core engine and telemetry options for Temporal. All
+    /// connections/clients created using it, and any workers created from them, will be associated
+    /// with the runtime.
+    /// </remarks>
+    public sealed class Runtime
+    {
+        /// <summary>
+        /// Current version of this SDK.
+        /// </summary>
+        public const string Version = "0.1.0-alpha1";
```

Comment:
Is this intentionally hardcoded here?
Is there any duplicate definition of this version string?

Reply:
Yes it is intentionally hardcoded and is the only place it is. I think it also may be duplicated in the `csproj` file at publish time, but I don't think it's reliable to retrieve that at runtime from the DLL itself so I put it here. Many other SDKs have this too (needed for creating Rust client).